tests/std/test.lst

```diff
@@ -340,3 +340,4 @@ tests\VSO_0849827_multicontainer_emplace_hint_position
 tests\VSO_0938757_attribute_order
 tests\VSO_0961751_hash_range_erase
 tests\VSO_0971246_legacy_await_headers
+tests\VSO_1062649_overaligned_function

```

Comment:
There's also VSO-157296  , but I didn't like it as there are too few digits.
It would be first GH- test, still want GH-690 name?

Reply:
> There's also [VSO-157296](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/157296) , but I didn't like it as there are too few digits.

We introduce leading zeroes so all bug numbers in `VSO_XXXX` test directories are 7 digits.

> It would be first GH- test, still want [GH-690](https://github.com/microsoft/STL/issues/690) name?

It would be the third GH test:

https://github.com/microsoft/STL/blob/c25d0a0d2242b866eef8af1a6704e21977269a9a/tests/std/test.lst#L158-L159

We prefer to use GH or DevCom numbers when possible since those are publicly visible bug databases.

Reply:
renamed